giskard_vision/core/detectors/metrics.py

```diff
@@ -0,0 +1,32 @@
+from abc import abstractmethod
+
+import numpy as np
+
+
+class MetricBase:
+    name: str
+    type_task: str
+
+    @abstractmethod
+    def get(self, pred, truth): ...
```

Comment:
This is definitely to remove, it conflicts with `giskard_vision/core/tests/base.py::MetricBase`.

so far we have relied on the module the metric is defined in to know it's type. I would keep the task_type an attr of the detector, and based on it, would should wether to use an MSE or Accuracy